guides/source/command_line.md

```diff
@@ -65,11 +65,12 @@ $ bin/rails server
 => Booting Puma
 => Rails 5.0.0 application starting in development on http://0.0.0.0:3000
 => Run `rails server -h` for more startup options
-=> Ctrl-C to shutdown server
-Puma 2.15.3 starting...
-* Min threads: 0, max threads: 16
+Puma starting in single mode...
+* Version 3.0.2 (ruby 2.3.0-p0), codename: Plethora of Penguin Pinatas
+* Min threads: 5, max threads: 5
```

Comment:
I believe so, but feel free to start a server yourself and compare. :)

I do, however, know that the "=> Ctrl-C to shutdown server" previously emitted by Rails is gone.